ui/brave_custom_notification/DEPS

```diff
@@ -0,0 +1,18 @@
+include_rules = [
+  "+chrome/browser/profiles",
```

Comment:
this is not allowed. If components depends on `//brave/ui/*` then `//brave/ui/*` cannot depend on `//chrome/*`